scripts/generate-ckpt.sh

```diff
@@ -7,14 +7,18 @@ usage() {
     echo ""
     echo "Options"
     echo "  --help -h  : Display this message"
-    echo "  -n <n>     : Number of harts"
     echo "  -b <elf>   : Binary to run in spike"
     echo "  -p <pc>    : PC to take checkpoint at [default 0x80000000]"
     echo "  -i <insns> : Instructions after PC to take checkpoint at [default 0]"
+    echo "  -o <out>   : Output directory to store the checkpoint in. [default <elf>.<randomnum>.loadarch]"
+    echo "  -v         : Verbose"
+    echo ""
```

Comment:
Why this space here?

Reply:
It separates out sections of commands. Maybe I should just write that things are mutually exclusive within the next two sections (DTB overrides any individual parameters given).